grpc/worker/worker.proto

```diff
@@ -32,3 +34,28 @@ message HealthCheckRequest {
 message HealthCheckResponse {
     string worker_addr = 1;
 }
+
+message ProviderInfo {
+    string name = 1;
+    int32 replica_id = 2;
+    string upstream_url = 3;
+    string provider_method = 4;
+    string status = 5;
+}
+
+message ProvidersResponse {
+    repeated ProviderInfo providers = 1;
+}
+
+message AddResourceRequest {
+    string name = 1;
+    string upstream_url = 2;
+    string provider_method = 3;
```

Comment:
These are grpc models, if you add models here, you should update the grpc python binding code too.
AND I SAID BEFORE, there is no "add resource" method in api definition. Have you read the review before: https://github.com/ProjectSoftoid/Kagami/pull/17#discussion_r1860040069 ?